## Fine-mapping of Prostate Cancer Susceptibility Loci in a Large Meta-Analysis Identifies Candidate Causal Variants

#### Tokhir Dadaev<sup>1,+</sup>, Edward J. Saunders<sup>1,+</sup>, Rosalind A. Eeles<sup>1,137,++</sup>, Zsofia Kote-Jarai<sup>1,++</sup>, *et al.*


### **Abstract**

Prostate cancer is a polygenic disease with a large heritable component. A number of common, low penetrance prostate cancer risk loci have been identified through GWAS. 

We applied the Bayesian multivariate variable selection algorithm JAM to finemap 84 prostate cancer susceptibility loci, using summary data from a large European ancestry meta-analysis.

We observed evidence for multiple independent signals at 12 regions and 99 risk signals overall. Only 15 original GWAS tag SNPs remain among the catalogue of candidate variants identified; the remainder are replaced by more likely candidates. Biological annotation of our credible set of variants indicates significant enrichment within promoter and enhancer elements, and transcription factor binding sites including AR, ERG and FOXA1. In 40 regions at least one variant is co-localised with an eQTL in prostate cancer tissue.

The refined set of candidate variants substantially increase the proportion of familial relative risk explained by these known susceptibility regions, which highlights the importance of fine-mapping studies and has implications for clinical risk profiling.

----------------

<sup>+</sup> These authors contributed equally to this work.  
<sup>++</sup> These authors joint senior authors.  
<sup>1</sup> The Institute of Cancer Research, London, SW7 3RP, UK.   
<sup>137</sup> Royal Marsden NHS Foundation Trust, London, SW3 6JJ, UK.   

----------------

<a href="https://www.nature.com/articles/s41467-018-04109-8" target="_blank">Nature Communications</a>   
<a href="https://pubmed.ncbi.nlm.nih.gov/29892050/" target="_blank">PMID: 29892050</a>
